pod install [!] Error installing SentryReactNative

OS:
- [ ]  Windows
- [X]  MacOS
- [ ]  Linux

_Platform:_
- [X]  iOS
- [ ]  Android

Output of `node -v && npm -v && npm ls --prod --depth=0`

```
v10.15.0
6.5.0
app@1.0.5 /Users/user/Desktop/app
├── apollo-boost@0.1.23
├── apollo-link@1.2.6
├── apollo-link-context@1.0.12
├── apollo-link-error@1.1.5
├── apollo-link-http@1.5.9
├── date-fns@1.30.1
├── graphql@14.0.2
├── graphql-tools@4.0.3
├── i18next@13.0.1
├── UNMET PEER DEPENDENCY react@16.7.0
├── react-apollo@2.3.3
├── react-i18next@8.4.0
├── react-native@0.57.8
├── react-native-actionsheet@2.4.2
├── react-native-amplitude-analytics@0.2.6
├── react-native-animatable@1.3.0
├── react-native-calendar-events@1.6.4
├── react-native-dotenv@0.2.0
├── react-native-gesture-handler@1.0.11
├── react-native-image-header-scroll-view@0.8.2
├── react-native-keyboard-aware-scroll-view@0.8.0
├── react-native-keychain@3.0.0
├── react-native-linear-gradient@2.5.3
├── react-native-modal@7.0.2
├── react-native-navigation-directions@1.0.3
├── react-native-progress@3.5.0
├── react-native-sentry@0.41.1
├── react-native-snap-carousel@3.7.5
├── react-native-star-rating@1.1.0
├── react-native-tab-view@1.3.1
├── react-native-text-input-mask@0.8.0
├── react-native-vector-icons@6.1.0
├── react-navigation@3.0.8
├── UNMET PEER DEPENDENCY styled-components@4.1.3
├── uuid@3.3.2
└── validator@10.10.0

npm ERR! peer dep missing: react@16.6.3, required by react-native@0.57.8
npm ERR! peer dep missing: styled-components@^2.0.0 || ^3.0.2, required by jest-styled-components@6.3.1
npm ERR! extraneous: apollo-link@1.2.3 /Users/user/Desktop/app/node_modules/apollo-link-state/node_modules/apollo-link
npm ERR! invalid: xcode@1.0.1-dev /Users/user/Desktop/app/node_modules/@sentry/wizard/node_modules/xcode
npm ERR! extraneous: uuid@3.0.1 /Users/user/Desktop/app/node_modules/@sentry/wizard/node_modules/uuid
```

Config:

```
Sentry.config('https://...@sentry.io/...', {
....
}).install()
```



---
I have following issue:

I get an error "Could not find remote branch v0.41.1 to clone." when I use Pods, because package.json version - 0.41.1 and last version in repo - 0.39.1.

I fixed it at myself by installing version 0.39.1 in the package.json

**Steps to reproduce:**
- yarn add react-native-sentry
- add to Podfile -> `pod 'SentryReactNative', :podspec => '../node_modules/react-native-sentry/SentryReactNative.podspec'`
- pod install

**Actual result:**
```
[!] Error installing SentryReactNative
[!] /usr/bin/git clone https://github.com/getsentry/react-native-sentry.git /var/folders/vy/wpz782fx7pj_d3qtfdwnbhgw0000gn/T/d20190211-80488-e0n69x --template= --single-branch --depth 1 --branch v0.41.1

Cloning into '/var/folders/vy/wpz782fx7pj_d3qtfdwnbhgw0000gn/T/d20190211-80488-e0n69x'...
warning: Could not find remote branch v0.41.1 to clone.
fatal: Remote branch v0.41.1 not found in upstream origin
```

**Expected result:**
- Success

--- a/SentryReactNative.podspec
+++ b/SentryReactNative.podspec
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.summary        = 'Official Sentry SDK for react-native'
   s.author         = 'Sentry'
   s.homepage       = "https://github.com/getsentry/react-native-sentry"
-  s.source         = { :git => 'https://github.com/getsentry/react-native-sentry.git', :tag => "v#{s.version}"}
+  s.source         = { :git => 'https://github.com/getsentry/react-native-sentry.git', :tag => "#{s.version}"}
 
   s.ios.deployment_target = "8.0"
   s.tvos.deployment_target = "9.0"

